ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -114,13 +114,14 @@ public struct AvatarGroup: View, ConfigurableTokenizedControl {
         let hasOverflow: Bool = overflowCount > 0
         let isStackStyle = tokens.style == .stack
 
-        let interspace: CGFloat = tokens.interspace
-        let ringOuterGap: CGFloat = tokens.ringOuterGap
-        let ringGapOffset: CGFloat = ringOuterGap * 2
-        let ringOffset: CGFloat = tokens.ringThickness + tokens.ringInnerGap + tokens.ringOuterGap
-
         let overflowAvatar = createOverflow(count: overflowCount)
         let imageSize: CGFloat = overflowAvatar.contentSize
+
+        let interspace: CGFloat = tokens.interspace
+        let avatarTokens = overflowAvatar.tokens
```

Comment:
These tokens might vary depending on the style of the avatars.
Overflow is supposed to be the last one, but the other ones should could get different tokens if those are overridden and we're retrieving the resolved ones for each avatar.

Reply:
Do we let avatars have different styles and sizes in the Avatar Group? Either way, the old way we got the tokens would've had that same problem.

Reply:
We don't have any way of setting different avatars through the group with different styles.
But the overflow avatar is an exception to that. It's a different style from the remainder of the avatars.
And I agree that this was gap we currently have. Probably not exposed so far as tokens were not overridden differently across styles.

Reply:
Interestingly, we shouldn't need any of these values unless `avatarViews.count > 0`, so we could always use the tokens from `avatarViews.first`, right? No need to leverage the overflow avatar which, as mentioned by @rdeassis, could easily be different than the rest at some point.